spec/controllers/post_spec.rb

```diff
@@ -4,8 +4,8 @@
   describe "GET #index" do
     it "responds successfully with an HTTP 200 status code" do
       get :index
+
       expect(response).to be_success
```

Comment:
probably should have a different description, since it's not checking for a '200' anymore, but 2xx. I wouldn't be opposed to just the `have_http_status(200)`, since that's always the code we would want in this case.